dist/marionette.carpenter.js

```diff
@@ -2623,9 +2623,9 @@ define('templates/row',[],function(){
     
       _print(_safe('\n'));
     
-      _ref = this.columns;
-      for (_i = 0, _len = _ref.length; _i < _len; _i++) {
-        column = _ref[_i];
+      ref = this.columns;
+      for (i = 0, len = ref.length; i < len; i++) {
+        column = ref[i];
```

Comment:
Were these edits made directly to `dist/marionette.carpenter.js`? That's the distribution file, the result of running a grunt task. Any edits here will be wiped out the next time someone builds the library for distribution.